src/fides/lib/oauth/schemas/user_permission.py

```diff
@@ -1,22 +0,0 @@
-from typing import List
-
-from fides.lib.schemas.base_class import BaseSchema
-
-
-class UserPermissionsCreate(BaseSchema):
-    """Data required to create a FidessUserPermissions record."""
-
-    scopes: List[str]
-
-
-class UserPermissionsEdit(UserPermissionsCreate):
-    """Data required to edit a FidesUserPermissions record."""
-
-    id: str
-
-
-class UserPermissionsResponse(UserPermissionsCreate):
-    """Response after creating, editing, or retrieving a FidesUserPermissions record."""
-
-    id: str
-    user_id: str
```

Comment:
We had two user permissions schemas, one brought back from fideslib.  Having two sets was inadvertently causing a bug where there was no validation on scopes when editing user permissions.  I've removed one set here.